### PR TITLE
#1268 add Composite Entity pattern

### PR DESCRIPTION
**Pull request by zWeBrain**
* Resolves issue [#1268 ](https://github.com/iluwatar/java-design-patterns/issues/1268)
* The idea of the pattern is to take advantage of a composite entity to manage multiple dependency objects and adjust the degree of granularity between objects. And it should be noted that the lifetime of dependency objects depends on the coarse-grained object.
* Additionally, this pattern is common for Entity Beans representing persistent data maintained in a database.

### CODE REVIEW

File: composite-entity/src/test/java/com/iluwatar/compositeentity/PersistenceTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Date: 4/17/21 - 11:45 AM
+ *
+ * @author zwebrain
+ */
```
Comment (ohbus) [inline composite-entity/src/test/java/com/iluwatar/compositeentity/PersistenceTest.java:11]: Please remove the author tag.

We can use git blame to know these metadata and avoid putting these to code

**Review by ohbus — CHANGES_REQUESTED** [review]
LGTM, take a look at the comments before we merge this PR

File: composite-entity/src/test/java/com/iluwatar/compositeentity/PersistenceTest.java (hunk 1 above)
Comment (zWeBrain) [reply composite-entity/src/test/java/com/iluwatar/compositeentity/PersistenceTest.java]: I'm sorry. I have removed it.

**Review by ohbus — APPROVED** [review]
LGTM

**Comment by ohbus** [comment]
Thank you so much @zWeBrain for your contribution! ㊗️ 

@all-contributors please add @zWeBrain  for code

**Pull request merged**